### PR TITLE
(PC-12072) fix(NoFavoritesResults): responsive for web orientation landscape

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-12072

## Checklist

I have:

- [ ] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [ ] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome responsive iPhone 5 | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|     | image| image| image |                  |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/favorites/components/NoFavoritesResult.tsx [hunk 1]
```diff
@@ -12,12 +12,15 @@ export const NoFavoritesResult = () => {
 
   return (
     <Container>
+      <Spacer.TabBar />
```
Comment (BenStirrup) [inline src/features/favorites/components/NoFavoritesResult.tsx:15]: pk tabbar ici dspécifiquement ?

Comment (kopax-polyconseil) [reply src/features/favorites/components/NoFavoritesResult.tsx:15]: Bonne question, le height de ce composant m'arrangeait bien, plutôt que d'en créer un nouveau (et CustomHeightSpacer n'est pas exporté.), que recommandes-tu ?

Comment (BenStirrup) [reply src/features/favorites/components/NoFavoritesResult.tsx:15]: on peut pas juste centrer le contenu ?

Comment (kopax-polyconseil) [reply src/features/favorites/components/NoFavoritesResult.tsx:15]: @BenStirrup , quand on réduit la hauteur du viewport, l'absence de spacing top fait que le contenu se ressert et fini par passer en dessous du header.

**Review by BenStirrup — APPROVED** [review]

**Review by BenStirrup — APPROVED** [review]

**Pull request merged**